oggsound/include/OgreOggISound.h

```diff
@@ -1,14 +1,14 @@
 /**
 * @file OgreOggISound.h
 * @author  Ian Stangoe
-* @version 1.26
```

Comment:
ugh.. you opened a can of worms there..

All this boilerplate code on top is mostly nonsense:
- nobody cares about @file
- a single @author is wrong
- @version is better tracked in cmake
- for License it is better to reference the top-level file and only leave a SPDX identifier here. See the Ogre Coding Guidelines: https://github.com/OGRECave/ogre/blob/master/Docs/CodingStandards.md#top-level-organisation-issues and https://spdx.dev/ids/

Reply:
but I guess it is too much hassle to remove the non-doxygen warning related changes now, so we should go with this as is?

Reply:
I think it should go as is to respect the original author mostly.

But if you want, I can remove the @file and @version

About the license the problem is that I don't know if the text provided is MIT or other license, sound like MIT...

Reply:
> I think it should go as is to respect the original author mostly.

I keep the original author in the "Copyright (c)" line of the [LICENSE](https://github.com/OGRECave/ogre-audiovideo/blob/master/oggsound/LICENSE) file for this reason.
Oggsound is MIT as you assumed, but ogrevideo is BSD-3

> But if you want, I can remove the @file and @Version

yes please